pages/software/python.md

```diff
@@ -0,0 +1,457 @@
+# Python Options
+
+There are many options for using Python and its derivates on CCR's systems.  CCR provides, at most, two versions of Python and two versions of Anaconda as well as SciPy (Scientific Python libraries) via it's modules system.  These are updated once per year with the oldest version being rotated out.  Users will see a warning when a module is being deprecated giving you advance notice to update your workflow.  
```

Comment:
Given that python2 is no longer supported, we'll likely only be providing python3. So perhaps here we just say CCR provides Python3, Anaconda, and SciPy via it's module systems.

Reply:
fixed